crates/agglayer-config/src/tracing.rs

```diff
@@ -3,24 +3,32 @@ use std::{fmt::Display, path::PathBuf};
 use serde::{Deserialize, Deserializer, Serialize};
 use tracing_subscriber::{fmt::writer::BoxMakeWriter, EnvFilter};
 
-/// The log configuration.
+/// The tracing configuration.
 #[derive(Serialize, Deserialize, Debug, Clone, Default, PartialEq, Eq)]
 #[serde(rename_all = "kebab-case")]
-pub struct Log {
+pub struct TracingConfig {
     /// The `RUST_LOG` environment variable will take precedence over the
-    /// configuration log level.
+    /// configuration tracing level.
     #[serde(default)]
-    pub level: LogLevel,
+    pub level: TracingLevel,
     #[serde(default)]
-    pub outputs: Vec<LogOutput>,
+    pub outputs: Vec<TracingOutput>,
     #[serde(default)]
-    pub format: LogFormat,
+    pub format: TracingFormat,
```

Comment:
This format represents the format of the `logs` exposed by the agglayer-node, I don't understand why we rename this to `TracingFormat`